src/containers/AssignmentTexterContact.jsx

```diff
@@ -211,7 +211,7 @@ export class AssignmentTexterContact extends React.Component {
       snackbarError,
       snackbarActionTitle,
       snackbarOnTouchTap,
-      optOutMessageText: "I'm opting you out of texts immediately. Have a great day.",
+      optOutMessageText: "I'm opting you out of texts immediately. Have a great day." || "",
```

Comment:
What does this change do?  the hard-coded string will never evaluate to false....